web/assets/styles/common.scss

```diff
@@ -1,217 +1,211 @@
 html,
 body {
-  background: $body-bg;
-  height: 100%;
+    background: var(--body-bg);
+    height: 100%;
 }
 
 .snap-content.image-bg {
-  //background-image: url("//source.unsplash.com/random");
-  background-repeat: no-repeat;
-  background-position: center center;
-  background-attachment: fixed;
-  -webkit-background-size: cover;
-     -moz-background-size: cover;
-       -o-background-size: cover;
-          background-size: cover;
-
-  body {
-    background: inherit;
-    overflow: auto;
-  }
+    //background-image: url("//source.unsplash.com/random");
+    background-repeat: no-repeat;
+    background-position: center center;
+    background-attachment: fixed;
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
+    background-size: cover;
+    body {
+        background: inherit;
+        overflow: auto;
+    }
 }
 
 .main.container {
-  background: $body-bg;
-  height: 100%;
-  position: absolute;
+    background: var(--body-bg);
+    height: 100%;
+    position: absolute;
 }
 
 h1,
 h2,
 h3,
 h4,
 h5 {
-  font-weight: 300;
-  margin: 10px 0;
+    font-weight: 300;
+    margin: 10px 0;
 }
 
 h1,
 h2,
 h3.add-widgets {
-  color: $widget-text-color;
+    color: var(--widget-text-color);
```

Comment:
Overrode it [here](https://github.com/kubawolanin/org.openhab.ui.habpanel/blob/c9ac71601a2302fcc975252cbf12ce74b8fb5eb0/web/assets/styles/themes/madras.css#L43).